x/epoching/keeper/msg_server.go

```diff
@@ -242,7 +243,7 @@ func (ms msgServer) WrappedCancelUnbondingDelegation(goCtx context.Context, msg
 		)
 	}
 
-	blockHeight := uint64(ctx.BlockHeader().Height)
+	blockHeight := uint64(ctx.HeaderInfo().Height)
```

Comment:
What is the difference here? Just curious.

Reply:
`ctx.BlockHeader()` returns `ctx.header` which is deprecated. We should use `HeaderInfo` instead. See https://github.com/cosmos/cosmos-sdk/blob/698627016d64dbf33bea01837364c5a6e55a1ee3/types/context.go#L45